app/src/main/java/edu/agh/susgame/front/settings/Configuration.kt

```diff
@@ -1,10 +1,10 @@
 package edu.agh.susgame.front.settings
 
 object Configuration {
```

Comment:
Hmm mamy efektywnie teraz 2 configi w aplikacji. Fajnie byłoby to połączyć z `edu.agh.susgame.front.Config`. Może by przerobić tę klasę w np. taki sposób:
```
class GameConfig(
    val MaxPinLength: Int,
    val gameTimeMinutes: GameTimeMinutes,
    val playersPerGame: PlayersPerGame,
) {
    class GameTimeMinutes(
        val Min: Int,
        val Max: Int,
    )

    class PlayersPerGame(
        val Min: Int,
        val Max: Int,
    )
}
```
i wrzucić jako pole do `edu.agh.susgame.front.util.AppConfig`. Można by w sumie usunąć cały ten plik i wszystko przenieść do wnętrza `AppConfig`